core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1636,6 +1638,13 @@ private void cancelPositionObserver() {
         }
     }
 
+    private void addPlayableToQueue(Playable playable) {
+        if (playable instanceof FeedMedia) {
+            FeedItem feedItem = DBReader.getFeedItem(((FeedMedia) playable).getId());
+            DBWriter.addQueueItem(this, feedItem);
```

Comment:
Sorry for taking so long to respond.

If you use `addQueueItem(this, false, false, ((FeedMedia) playable).getId())` instead, it does not need to do the database query that retrieves the item.

By the way, this also fixes #3277 :) :+1:

Reply:
Good call, I updated the pr